mix_helpers.exs

```diff
@@ -252,13 +252,28 @@ defmodule Mix.Appsignal.Helper do
   end
 
   defp download_options do
```

Comment:
If possible, can you add some tests for this in the mix helpers test file?

Reply:
It's a bit tricky to test this file. A solution would be to make `download_options` a public method. I'm not sure if that's worth it.